File: Fundamentais/Javascript/1. Troca de valores de 2 variaveis.js
// Algoritmo Fundamental Nº 1
// Descrição : Troca dos valores de duas variáveis.
// Problema  : Dadas duas variáveis, trocar os valores atribuídos a elas.
// Aplicações: Algoritmos de ordenação.

var a = prompt("Digite o valor de A:");
var b = prompt("Digite o valor de B:");

var aux = a;
a = b;
b = aux;

document.write("A = " + a + "<br>");
document.write("B = " + b + "<br>");

File: Fundamentais/Python/1. Troca de valores de 2 variaveis.py
# Algoritmo Fundamental Nº 1
# Descrição : Troca dos valores de duas variáveis
# Problema  : Dadas duas variáveis, trocar os valores atribuídos a elas.
# Aplicações: Algoritmos de ordenação.

a = input("Digite o valor de A: ")
b = input("Digite o valor de B: ")

aux = a
a = b
b = aux

print("A =", a)
print("B =", b)
